### PR TITLE
Use latest .browserslistrc, package updates and tidy-up

### PR DESCRIPTION
**Pull request by colinrotherham**
Hello 👋 

Mainly a spring clean in this PR, but covers a few things:

**1. Removed old/used Sass/PostCSS tools from `./design-system`**
Instead, all packages now use the same consistent config

**2. Update all packages using Lerna**
No more package version hoist mismatches or warnings

**3. Removed `cssimport()` from Design System and import via _node_modules_**
Enables the _localhost_ Design System to see your unpublished package changes

You'll see this symlink messages now at install time:

```
lerna WARN EREPLACE_EXIST @coopdigital/foundations-buttons is already installed for design-system. Replacing with symlink...
lerna WARN EREPLACE_EXIST @coopdigital/foundations-colors is already installed for design-system. Replacing with symlink...
lerna WARN EREPLACE_EXIST @coopdigital/foundations-forms is already installed for design-system. Replacing with symlink...
```

**4. Swapped out some mixed PostCSS configs with a nice new one**
i.e. Why do packages use `postcss-nesting` but Design System uses `postcss-nested`?

**5. Added @dalecreativ's commits for the new Browserlist configs 🎉**
Why? We noticed `last 2 versions` added last two of everything (Windows Mobile IE, Blackberry, and other super-old browsers)

### CODE REVIEW

File: design-system/gulpfile.js [hunk 1]
```diff
@@ -25,7 +19,7 @@ const src = 'src/';
 const dest = 'build/';
 
 const src_paths = {
-  css: src + '_css/**/*.{pcss,css}',
+  css: src + '_css/*.{pcss,css}',
```
Comment (colinrotherham) [inline design-system/gulpfile.js:22]: We only need to compile the top-level CSS, the nested ones are loaded via `@import`

File: design-system/gulpfile.js [hunk 2]
```diff
@@ -44,15 +38,6 @@ const dest_paths = {
 const settings = {
   css: {
     outputStyle: 'compressed',
-    includePaths: [
```
Comment (colinrotherham) [inline design-system/gulpfile.js:47]: We didn't need these as we can specify the paths in the CSS instead (and symlinks work!)

**Before**
```css
/* Foundations */
@import "normalize.css/normalize.css";
@import "foundations-vars/src/vars.pcss";
@import "foundations-global/src/global.pcss";
```

**After**
```css
/* Foundations */
@import "normalize.css";
@import "@coopdigital/foundations-vars";
@import "@coopdigital/foundations-global";
```

File: design-system/gulpfile.js [hunk 3]
```diff
@@ -125,18 +99,7 @@ function css() {
...
-        [
-          postcssCustomMedia(),
-          postcssCustomProperties(),
-          postcssNesting(),
-          postcssCalc()
-        ]
-      )
-    )
-    .pipe(autoprefixer())
+    .pipe(postcss())
```
Comment (colinrotherham) [inline design-system/gulpfile.js:102]: If we leave the config out, it'll magically find **postcss.config.js** instead 🎉

File: design-system/package.json [hunk 4]
```diff
@@ -31,58 +31,43 @@
...
     "dist": "echo Design System - no need to build",
     "server": "gulp server"
   },
-  "browserslist": [
-    "> 5%",
-    "last 2 versions"
-  ],
   "devDependencies": {
-    "postcss-custom-properties": "^9.0.2",
+    "autoprefixer": "^9.8.5",
```
Comment (colinrotherham) [inline design-system/package.json:65]: Best if we keep development tools in `devDependencies`

File: design-system/src/_css/main.css [hunk 5]
```diff
@@ -3,34 +3,34 @@
...
-@import "foundations-global/src/global.pcss";
-@import "foundations-typography/src/typography.pcss";
-@import "foundations-grid/src/grid.pcss";
-@import "foundations-colors/src/colors.pcss";
-@import "foundations-buttons/src/buttons.pcss";
-@import "foundations-forms/src/forms.pcss";
-@import "foundations-tables/src/tables.pcss";
-@import "foundations-layout/src/layout.pcss";
+@import "normalize.css";
+@import "@coopdigital/foundations-vars";
```
Comment (colinrotherham) [inline design-system/src/_css/main.css:7]: If we use the proper package names, Lerna symlinks work 👍

File: postcss.config.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  map: {
+    inline: false
```
Comment (colinrotherham) [inline postcss.config.js:3]: Unlike the **Design System** which uses `gulp-sourcemaps` we do need to specific a map config here when building the individual `./packages/*` files via _postcss-cli_.

(Compare with `./design-system/postcss.config.js`)

File: design-system/src/_css/main.css [hunk 7]
```diff
@@ -14,18 +14,19 @@
 @import "@coopdigital/foundations-tables";
 @import "@coopdigital/foundations-layout";
 
+/* TODO: Fix components with typos in published path */
```
Comment (colinrotherham) [inline design-system/src/_css/main.css:17]: Sadly a couple of packages had typos in their paths, we can revert this after the next `npm publish`

File: packages/component-card--product/.browserlistrc [hunk 8]
```diff
@@ -1,3 +1,3 @@
-> 1%
-last 3 versions
+> 2% in GB
+last 2 versions and not <= 0.2% in GB
```
[the remark this answers is not in the document]
Comment (froddd) [reply packages/component-card--product/.browserlistrc:2]: As discussed, these should not be package-specific and can be covered by root config

[the remark this answers is not in the document]
Comment (colinrotherham) [reply packages/component-card--product/.browserlistrc:2]: Yeah would love to ❤️ 

Sounds like one for another PR after this lands?

[the remark this answers is not in the document]
Comment (froddd) [reply packages/component-card--product/.browserlistrc:2]: Happy with that!

**Pull request merged**